Assets/Scripts/ROS Scripts/ROS Connections v2/ROS Connection Interfaces.cs

```diff
@@ -32,15 +32,21 @@ public interface ROSDroneConnectionInterface
     void LandDrone();
     void FlyHome();
 
+    // Close ROS Connection
```

Comment:
Please use the <summary> tag.

Also if its documented here, you usually don't need to document in the child class. (or vice versa but this is less work as there are less parent classes)